docs/make.jl

```diff
@@ -13,27 +13,30 @@ format = Documenter.HTML(edit_link = "source",
 
 makedocs(modules  = [Images, ImageCore, Colors, ColorTypes, FixedPointNumbers, ImageAxes,
                     ImageFeatures, ImageFiltering, ImageMetadata, ImageContrastAdjustment,
+                    ImageDistances, ImageQualityIndexes, MosaicViews, TestImages, FileIO,
                     ImageSegmentation, ImageTransformations, PaddedViews, ImageMorphology],
          format   = format,
          sitename = "JuliaImages",
          pages    = ["Home" => "index.md",
-                     "install.md",
-                     "Manual" => Any[
-                         "quickstart.md",
-                         "arrays_colors.md",
-                         "conversions_views.md",
-                         "indexing.md",
-                         "imageaxes.md",
-                         "imagefiltering.md",
-                         "imagemetadata.md",
-                         "imagesegmentation.md",
-                         "imagetransformations.md",
-                         "imagefeatures.md",
-                         "troubleshooting.md",
+                     "Tutorials" => Any[
+                         joinpath("tutorials", "install.md"),
+                         joinpath("tutorials", "quickstart.md"),
+                         joinpath("tutorials", "arrays_colors.md"),
+                         joinpath("tutorials", "conversions_views.md"),
+                         joinpath("tutorials", "indexing.md")
+                     ],
+                     "Packages" => Any[
+                         "Introduction" => joinpath("pkgs", "index.md"),
+                         "ImageAxes.jl" => joinpath("pkgs", "axes", "index.md"),
+                         "ImageMetaData.jl" => joinpath("pkgs", "metadata", "index.md"),
+                         "ImageSegmentation.jl" => joinpath("pkgs", "segmentation", "index.md"),
```

Comment:
As you might have noticed, this nested folder structure contributes to a very long URL.

Reply:
As this PR shows, modularization makes it easier to access their own assets from "*.md". On the other hand, it makes it difficult to access the common assets, and `@example` and so on run in the directory "build". So I think the length of the local relative path is more important than the URL.
There is not much difference, though.

Reply:
BTW, the assets in "src" will be copied into the "build" before running doctests. So I don't think the "src/" is needed when loading an asset. That improves the consistency between `load` and `save`.